src/CommunityToolkit.Maui.Core/Essentials/FileFolderDialog.tizen.cs

```diff
@@ -0,0 +1,398 @@
+		{
+			LinearOrientation = LinearLayout.Orientation.Vertical,
+		};
+		content.Add(directoryScrollView);
+
+		UpdateDirectoryScrollView(initialFileFolderPath);
+
+		content.Add(new View
+		{
+			BackgroundColor = TColor.FromHex("#cccccc").ToNative(),
+			SizeHeight = 1.5d.ToPixel(),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		});
+
+		if (isFileSelectionMode)
+		{
+			var newFolderButton = new Button
+			{
+				Focusable = true,
+				Text = "New Folder",
+				TextColor = TColor.Black,
+				BackgroundColor = TColor.Transparent.ToNative(),
+				Margin = new Extents(margin1, margin1, 0, 0),
+				WidthSpecification = LayoutParamPolicies.MatchParent
+			};
+			newFolderButton.Clicked += async (sender, args) =>
+			{
+				try
+				{
+					var newDirectoryName = await new PromptPopup("New Folder Name", "").Open();
+					var newDirectoryPath = IsDirectory(selectedPath) ? selectedPath : Path.GetDirectoryName(selectedPath);
+					if (string.IsNullOrEmpty(newDirectoryPath) || string.IsNullOrEmpty(newDirectoryName))
+					{
+						return;
+					}
+					newDirectoryPath = newDirectoryPath.EndsWith(slashSymbol) ? newDirectoryPath : newDirectoryPath + slashSymbol;
+					if (CreateSubDirectory(newDirectoryPath + newDirectoryName))
+					{
+						UpdateDirectoryScrollView(selectedPath);
+					}
+				}
+				catch (Exception ex)
+				{
+					new Tizen.Applications.ToastMessage
+					{
+						Message = $"Failed to create new folder, {ex.Message}"
+					}.Post();
+				}
+			};
+			content.Add(newFolderButton);
+		}
+
+		if (isFileSelectionMode)
+		{
+			fileNameEntry = new Entry()
+			{
+				Text = selectedFileName,
+				Margin = new Extents(margin1, margin1, 0, 0),
+				WidthSpecification = LayoutParamPolicies.MatchParent
+			};
+			PropertyMap underline = new PropertyMap();
+			underline.Add("enable", new PropertyValue("True"));
+			fileNameEntry.Underline = underline;
+			fileNameEntry.PixelSize = 15d.ToPixel();
+			content.Add(fileNameEntry);
+		}
+
+		var hlayout = new View
+		{
+			Margin = new Extents(margin1, margin1, margin2, margin1),
+			Layout = new LinearLayout
+			{
+				VerticalAlignment = VerticalAlignment.Center,
+				HorizontalAlignment = HorizontalAlignment.End,
+				LinearOrientation = LinearLayout.Orientation.Horizontal,
+			},
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+			HeightSpecification = LayoutParamPolicies.WrapContent
+		};
+		content.Add(hlayout);
+
+		var cancelButton = new Button
+		{
+			Focusable = true,
+			Text = "Cancel",
+			TextColor = TColor.Black,
+			BackgroundColor = TColor.Transparent.ToNative(),
+		};
+		cancelButton.TextLabel.PixelSize = 15d.ToPixel();
+		cancelButton.SizeWidth = cancelButton.TextLabel.NaturalSize.Width + 15d.ToPixel() * 2;
+		cancelButton.Clicked += (s, e) => SendCancel();
+		hlayout.Add(cancelButton);
+
+		var okButton = new Button
+		{
+			Focusable = true,
+			Text = isFileSelectionMode ? "Save" : "OK",
```

Comment:
This string isn't localized... And there're others that I'll not comment to not spam the review